reference_entities/enrich_a_record_with_new_attribute_type.rst

```diff
@@ -0,0 +1,443 @@
+     *
+     * It returns the JSX View to display the field of the custom Record Value.
+     */
+    const View = ({
+      value,
+      onChange,
+      onSubmit,
+      canEditData,
+    }: {
+      value: Value;
+      onChange: (value: Value) => void;
+      onSubmit: () => void;
+      canEditData: boolean;
+    }) => {
+      if (!(value.data instanceof SimpleMetricData && value.attribute instanceof ConcreteSimpleMetricAttribute)) {
+        return null;
+      }
+
+      const onValueChange = (text: string) => {
+        const newData = denormalize(text);
+        if (newData.equals(value.data)) {
+          return;
+        }
+
+        const newValue = value.setData(newData);
+
+        onChange(newValue);
+      };
+
+      return (
+        <React.Fragment>
+          <input
+            id={`pim_reference_entity.record.enrich.${value.attribute.getCode().stringValue()}`}
+            autoComplete="off"
+            className={`AknTextField AknTextField--narrow AknTextField--light
+              ${value.attribute.valuePerLocale ? 'AknTextField--localizable' : ''}
+              ${!canEditData ? 'AknTextField--disabled' : ''}`}
+            value={value.data.normalize()}
+            onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+              onValueChange(event.currentTarget.value);
+            }}
+            onKeyDown={(event: React.KeyboardEvent<HTMLInputElement>) => {
+              if (Key.Enter === event.key) onSubmit();
+            }}
+            disabled={!canEditData}
+            readOnly={!canEditData}
+          />
+          <span>{value.attribute.unit.normalize()}</span>
+        </React.Fragment>
+      );
+    };
+
+    /**
+     * The only required part of the file: exporting the custom Record Value view.
+     */
+    export const view = View;
+
+3) Cell
+^^^^^^^
+
+The last part we need to do is to create the React component to be able to render a cell in the Record Grid.
+
+.. code-block:: javascript
+
+    import {NormalizedValue} from 'akeneoreferenceentity/domain/model/record/value';
+    import {CellView} from 'akeneoreferenceentity/application/configuration/value';
+    import {denormalize as denormalizeAttribute} from "custom/reference-entity/attribute/simple_metric";
+    import {NormalizedSimpleMetricAttribute} from "../attribute/simple_metric";
+    import {Column} from "akeneoreferenceentity/application/reducer/grid";
+
+    const memo = (React as any).memo;
+
+    /**
+     * Here we define the React Component as a function with the following props :
+     *    - the custom Record Value
+     *
+     * It returns the JSX View to display the cell of your custom Record Value in the grid.
+     */
+    const SimpleMetricCellView: CellView = memo(({column, value}: {column: Column, value: NormalizedValue}) => {
+      const simpleMetricData = denormalize(value.data);
+      const simpleMetricAttribute = denormalizeAttribute(column.attribute as NormalizedSimpleMetricAttribute);
```

Comment:
maybe we should denormalize the attribute before passing it to the cells because if every celle denormalize the attribute, it will be vénère